modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -128,6 +128,7 @@ interface DOMUtils {
     <K extends keyof HTMLElementTagNameMap>(name: K, attrs?: Record<string, string | boolean | number | null>, html?: string | Node | null): HTMLElementTagNameMap[K];
     (name: string, attrs?: Record<string, string | boolean | number | null>, html?: string | Node | null): HTMLElement;
   };
+  createTextNode: (data: string) => Text;
```

Comment:
Since DOMUtils is exposed, this should also have a changelog entry?

Reply:
changelog entry added